pkg/agent/flowexporter/connections/connections.go

```diff
@@ -23,12 +23,15 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/klog/v2"
 
+	"github.com/vmware/go-ipfix/pkg/registry"
```

Comment:
should be in the same group as "k8s.io/klog/v2"

Reply:
fixed, thanks!